rezervo/database/crud.py

```diff
@@ -372,3 +373,31 @@ def purge_slack_receipts(db) -> int:
     )
     db.commit()
     return row_count
+
+
+def get_community(db: Session, user_id: UUID) -> Community:
+    users = (
+        db.query(models.User)
+        .filter(models.User.id != user_id)
+        .order_by(models.User.name)
+        .all()
+    )
+    chain_users = (
+        db.query(models.ChainUser).filter(models.ChainUser.user_id != user_id).all()
+    )
+
+    user_to_chain_map: Dict[UUID, List[str]] = {}
+    for chain_user in chain_users:
+        user_to_chain_map.setdefault(chain_user.user_id, []).append(chain_user.chain)
```

Comment:
Have you considered using [`defaultdict`](https://docs.python.org/3/library/collections.html#collections.defaultdict)?

Reply:
(https://github.com/mathiazom/rezervo/pull/33/commits/9ff1777e9bf573505933d3c66796c7054cc9b942)

Reply:
Smart, did not know that was a thing